pipelines/internal/push-artifacts-to-cdn/README.md

```diff
@@ -15,3 +15,6 @@ Tekton Pipeline to push artifacts to CDN and/or Dev Portal
 | cgwSecret       | Env specific secret containing the content gateway credentials                         | No          | -                                                          |
 | taskGitUrl      | The url to the git repo where the release-service-catalog tasks to be used are stored  | Yes         | https://github.com/konflux-ci/release-service-catalog.git  |
 | taskGitRevision | The revision in the taskGitUrl repo to be used                                         | No          | -                                                          |
+
+## Changes in 0.0.2
+* Remove debug branch name
```

Comment:
I don't see any change in the pipeline definition

Reply:
fixed

Reply:
You should also remove the version change from the yaml file.

Reply:
fixed